### PR TITLE
Dice game example

### PR DESCRIPTION
**Pull request by marcdel**
Adds a dice game example in service of this documentation pull request: https://github.com/open-telemetry/opentelemetry.io/pull/2685

### CODE REVIEW

**Comment by benregn** [comment]
It's because Phoenix now vendors all the icons from https://heroicons.com/ for newly generated projects. I assume they could be trimmed back since probably only a handful of icons (maybe?) are used in this project 🙂

**Comment by tsloughter** [comment]
None should be used in this project, so they can all be removed.

**Comment by benregn** [comment]
@lessless I'm not the author of this PR but thanks for the suggestion. 

However in my opinion, an example using Phoenix is probably more helpful because it's used more in the wild.

**Review by tsloughter — APPROVED** [review]
I'll merge once the unnecessary files, like icons are removed.

**Comment by tsloughter** [comment]
@marcdel ping

**Comment by marcdel** [comment]
@tsloughter sorry, I totally spaced on this. I regenerated the app without all the html stuff which simplifies it significantly. Happy to add metrics examples as well, but probably as a separate PR.

File: examples/roll_dice/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+  def roll(conn, _params) do
+    send_resp(conn, 200, roll_dice())
+  end
+
+  defp roll_dice do
+    Tracer.with_span("roll_dice") do
+      roll = Enum.random(1..6)
+
+      Tracer.set_attribute(:roll, roll)
```
Comment (tsloughter) [inline examples/roll_dice/README.md:65]:
```suggestion
      Tracer.set_attribute(:"roll.value", roll)
```

**Review by tsloughter — CHANGES_REQUESTED** [review]
Looks good! Just had one change to match the other implementations of the example in other languages.

**Comment by tsloughter** [comment]
Thanks! Sounds good with having another PR.

**Comment by marcdel** [comment]
@tsloughter boop! rebased this guy, so should be good to go.

**Review by tsloughter — APPROVED** [review]

**Pull request merged**